### PR TITLE
[BCF-2705] Randomize addresses in forwarders tests

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

File: core/cmd/forwarders_commands_test.go [hunk 1]
```diff
@@ -76,7 +75,7 @@ func TestShell_TrackEVMForwarder(t *testing.T) {
 	set := flag.NewFlagSet("test", 0)
 	cltest.FlagSetApplyFromAction(client.TrackForwarder, set, "")
 
-	require.NoError(t, set.Set("address", "0x5431F5F973781809D18643b87B44921b11355d81"))
+	require.NoError(t, set.Set("address", utils.RandomAddress().Hex()))
```
Comment (george-dorin) [inline core/cmd/forwarders_commands_test.go:78]: Won't this create a new address?

Comment (cedric-cordenier) [reply core/cmd/forwarders_commands_test.go:78]: It does, yes -- is that a concern?

Comment (george-dorin) [reply core/cmd/forwarders_commands_test.go:78]: Isn't this supposed to be the same address as above? So it should be `address.Hex()`?

**Review by jmank88 — APPROVED** [review]

File: core/cmd/forwarders_commands_test.go (hunk 1 above)
Comment (george-dorin) [reply core/cmd/forwarders_commands_test.go:78]: Ahh, now I see that we are not testing for that, disregard

**Review by george-dorin — APPROVED** [review]

**Pull request merged**